packages/api-docs-builder/utils/getPropsFromComponentNode.ts

```diff
@@ -250,14 +266,13 @@ function getPropsFromVariableDeclaration({
         });
       }
     }
-
-    return null;
   }
 
   // handle component factories: x = createComponent()
   if (
     checkDeclarations &&
     node.initializer &&
+    !isStyledFunction(node) &&
```

Comment:
Before `getJSXLikeReturnValueFromFunction` did not recognize `React.ReactNode` as a valid JSX response, so `styled` components were ignored.

Now they are supported, but we probably don't want them to be so I added a test to filter them out.